### PR TITLE
Updated DCC's latest supported versions.

### PR DESCRIPTION
**Pull request by vazquea**
Update supported versions at https://developer.shotgridsoftware.com/cc3d2160/?title=Engine+Supported+Versions

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/1e5213f0059c24926176f0b20e2c8aef81df1649/index.html)

File: docs/_includes/tk-photoshopcc [hunk 1]
```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 — 24.1.1
```
Comment (julien-lang) [inline docs/_includes/tk-photoshopcc:1]: For Adobe, we need to decide whether we use year versions of version numbers.
As a user, I think you see the year's version first.
I suggest doing both: `2023 (v24.1.1)`

Could you align the same logic on oldest version / newest version, AE and PS?

File: docs/_includes/tk-nuke [hunk 2]
```diff
@@ -1 +1 @@
-11.0 - 13.1
\ No newline at end of file
+11.0 - 14.0
```
Comment (julien-lang) [inline docs/_includes/tk-nuke:1]: I think we said we support the newest version + 2 previous ones. So we should switch oldest supported from 11 to 12. right?

Comment (julien-lang) [reply docs/_includes/tk-nuke:1]: same for vred

**Review by julien-lang — CHANGES_REQUESTED** [review]

**Review by shaynacohen — APPROVED** [review]
The files updated are perfect :)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/57a75b9e7b63a3018cc06bc49e010bacb93afe67/index.html)

File: docs/_includes/tk-3dsmax [hunk 3]
```diff
@@ -1 +1 @@
-2018+
\ No newline at end of file
+2023
```
Comment (julien-lang) [inline docs/_includes/tk-3dsmax:1]: Do we only support one version? What happened to the "current + 2 previous ones" rule? Does that not apply here?

File: docs/_includes/tk-aftereffects [hunk 4]
```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 - 2023 (v23.2.1)
```
Comment (julien-lang) [inline docs/_includes/tk-aftereffects:1]: Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)

According to https://en.wikipedia.org/wiki/Adobe_After_Effects, it should be 2021 (v18.0)

File: docs/_includes/tk-alias [hunk 5]
```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 — 2023
```
Comment (julien-lang) [inline docs/_includes/tk-alias:1]: Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)

File: docs/_includes/tk-photoshopcc [hunk 6]
```diff
@@ -1 +1 @@
-2019 — 2022
\ No newline at end of file
+2019 — 2024 (v24.1.1)
```
Comment (julien-lang) [inline docs/_includes/tk-photoshopcc:1]: Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)

According to https://en.wikipedia.org/wiki/Adobe_Photoshop#2021_(version_22), it should be 2021 (v22)
?

File: docs/_includes/tk-vred [hunk 7]
```diff
@@ -1 +1 @@
-2020.3 — 2022
\ No newline at end of file
+2020.3 — 2023
```
Comment (julien-lang) [inline docs/_includes/tk-vred:1]: Ok for the new version but should we not increment our "oldest supported" version? (according to our "current + 2 previous ones" rule)

Does this apply here?

File: docs/_includes/tk-mari [hunk 8]
```diff
@@ -1 +1 @@
-4.0 — 4.7
\ No newline at end of file
+4.0 — 4.8
```
Comment (julien-lang) [inline docs/_includes/tk-mari:1]: I wonder how we apply the "current + 2 previous ones" rule for this one.
Any idea?

**Review by julien-lang — CHANGES_REQUESTED** [review]

File: docs/_includes/tk-mari (hunk 8 above)
Comment (vazquea) [reply docs/_includes/tk-mari:1]: Maybe supporting the previous 2 minors? Like 4.6 - 4.8

Comment (julien-lang) [reply docs/_includes/tk-mari:1]: @mathurf any advice?

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b5957fe03945dd9986d719cea63639251531ddee/index.html)

**Review by julien-lang — APPROVED** [review]
LGTM

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/b5957fe03945dd9986d719cea63639251531ddee/index.html)

**Pull request merged**